content/How to Make My First Gem - Timothy Peraza.md

```diff
@@ -0,0 +1,127 @@
+# **How To Make Your First Ruby Gem (a step-by-step guide)**
+
+It could be argued, that most developers who have worked with ruby on rails, might be interested in developing a specific gem to solve problems for their fellow developers. 
+
+However, it's not uncommon for developers to not know where to begin, or where a gem is created in the first place. It's also not uncommon for a developer to feel like this would be a difficult and complicated task. For that reason, the aim of this article is to explain how to develop your first gem, and also take a look at the different components involved.  
```

Comment:
```suggestion
However, it's not uncommon for developers to not know where to begin, or to know where a gem is created in the first place. It's also not uncommon for a developer to feel like this would be a difficult and complicated task. For that reason, the aim of this article is to explain how to develop your first gem, and also take a look at the different components involved.  
```